releasenotes/notes/fix-profiler-3-11-memory-leak-stack-collector-9131642011ee5ce8.yaml

```
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    profiling: This fix resolves an issue in Python 3.11 where a PyFrameObject strong reference count was not properly
+    decremented in the stack collector.
```

```suggestion
    decremented in the stack collector.
    
```
do we want a newline here?

---

I believe we've consistently had newlines at the end of release note files 😄